js/scripts.js

```diff
@@ -0,0 +1,36 @@
+var iPhone6S = new Phone("Apple", 2250, "silver"),
+    SamsungGalaxyS6 = new Phone("Samsung", 2000, "black"),
```

Comment:
variables names should start with small letter, first big letter is reserved only for constructor functions